### PR TITLE
Fix configured_repository_name check for Bazel 6

### PR DESCRIPTION
**Pull request by jmthvt**
Attempt to fix https://github.com/bazelbuild/rules_kotlin/issues/883

### CODE REVIEW

File: src/main/starlark/core/repositories/initialize.release.bzl [hunk 1]
```diff
@@ -96,7 +96,7 @@ def kotlin_repositories(
         if (criteria and compiler_release.version.startswith(criteria.prefix)) or (not selected_version and not criteria):
             selected_version = version
 
-    if configured_repository_name:  # without a repository name, no default kt_* rules repository is created.
+    if configured_repository_name != "":  # without a repository name, no default kt_* rules repository is created.
```
Comment (restingbull) [inline src/main/starlark/core/repositories/initialize.release.bzl:99]: Uuuuuh.... is this change in starlark?

Comment (SrodriguezO) [reply src/main/starlark/core/repositories/initialize.release.bzl:99]: Hmm the build failed with:
```
Caused by: java.lang.ClassCastException: class net.starlark.java.eval.NoneType cannot be cast to class java.lang.String (net.starlark.java.eval.NoneType is in unnamed module of loader 'app'; java.lang.String is in module java.base of loader 'bootstrap')
```

I think we need this instead?
```suggestion
    if configured_repository_name and configured_repository_name != "":  # without a repository name, no default kt_* rules repository is created.
```

**Comment by Clement-Jean** [comment]
Any update on this?

**Comment by Bencodes** [comment]
@Clement-Jean this shouldn't be needed anymore as long as you are using one of the newer releases.

**Pull request closed without merging**